ui/apps/platform/src/Containers/Policies/Wizard/Form/descriptors.tsx

```diff
@@ -733,6 +733,27 @@ export const policyConfigurationDescriptor: Descriptor[] = [
         })),
         canBooleanLogic: false,
     },
+    {
+        label: 'Allow Privilege Escalation',
```

Comment:
Red Hat style for labels is to use Sentence-case. (first word capitalized, remaining words lowercase.
```suggestion
        label: 'Allow privilege escalation',
```

Reply:
Done, thanks Van.